person.rb

```diff
@@ -2,14 +2,15 @@
 
 class Person < Nameable
   attr_accessor :name, :age
-  attr_reader :id
+  attr_reader :id, :rentals
 
   def initialize(age, name = 'Unknown', parent_permission: true)
     super()
     @id = Random.rand(1...1000)
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @rentals = []
   end
 
```

Comment:
- [ ] To implement the has-many side of the relationship, a person has-many rentals, method for adding rentals should be present, something like so: 
```ruby
  def add_rental(book, date)
    Rental.new(date, self, book)
  end
```
Please kindly fix that